prompts/deepset-conversational-agent-without-tools.md

```diff
@@ -0,0 +1,33 @@
+This prompt is intended to be used with a Haystack `Agent`, specificallly, a [`ConversationalAgent`](https://docs.haystack.deepset.ai/docs/agent#conversational-agent).
+
+To learn how to use this prompt and the `ConversationalAgent`, check out our [Building a Conversational Chat App Tutorial](https://haystack.deepset.ai/tutorials/24_building_chat_app).
+
+By default, the `ConversationalAgent` uses if you don't provide any tools. However, below is an example of how you might manually create a `PromptTemplate` with this prompt and use it with the ConversationalAgent.
+
+## How to use in Haystack
+
+1. Create a `PromptTemplate` that uses `deepset/conversational-agent-with-tools`:
+
+```python
+import os
+
+from haystack.nodes import PromptNode, PromptTemplate
+
+conversational_agent_without_tools_from_hub = PromptTemplate("deepset/conversational-agent-without-tools")
```

Comment:
Why do we need this `_from_hub` part? Is it to show that's it coming from the hub? Perhaps it is not needed.